Qpf/Macro/Comp.lean

```diff
@@ -317,6 +327,11 @@ def elabQpfComposition (view: QpfCompositionView) : CommandElabM Unit := do
         CurriedTypeFun $live_arity := 
       TypeFun.curried $F_internal_applied
 
+      $modifiers:declModifiers
+      instance : 
+        MvFunctor ($F_internal_applied) := 
+      by unfold $F_internal; infer_instance
```

Comment:
For now, I just ended up adding this because otherwise some of the tests would fail.
Which other instance do you mean in particular? I'm afraid I'm not familiar enough with the code yet to see the connection :sweat_smile:

Reply:
Ah, I'd misread the diff. I was referring to the following lines, not realizing that GitHub was omitting the few hundred lines in between.
```lean
let func ← synthMvFunctor F
let _ ← synthQPF F func
```

Please ignore my comment